scene-format.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
```

Comment:
Let's move this file to `proto/scene.proto`, so we don't have to run protoc in the repo's root.